### PR TITLE
Sulphuric Acid Tanks (again)

### PR DESCRIPTION
**Pull request by Apogee-dev**
bigsacidtank

Take 2 this time. Adds sulphuric acid tanks, which can be ordered from Cargo. They come in a locked crate (science or engineering access) and cost the same as the other orderable reagent tanks. They also refill soldering irons instantly if you use the iron directly on the tank.

:cl:
 * rscadd: Added sulphuric acid tanks to the cargo catalogue.

### CODE REVIEW

**Comment by Optimism333** [comment]
It is the sovereign right of every Grey citizen to own at least one of these tanks for personal consumption.

**Comment by Eneocho** [comment]
On my way to make 10 nitrobombs. Based.

**Comment by Eneocho** [comment]
I think for consistency the sacid tank should come in a large box just like the welder fuel tank instead of a renamed shardcrate. You can "acquire" sacid from engi and chem, so it ain't like the shardcrate will deter sacid memery.

**Comment by Apogee-dev** [comment]
> 
> 
> I think for consistency the sacid tank should come in a large box just like the welder fuel tank instead of a renamed shardcrate. You can "acquire" sacid from engi and chem, so it ain't like the shardcrate will deter sacid memery.

Locked cargo crates are never a deterrent to a determined player, but they at least require you to put in a token effort to open them (just like it takes at least a token effort to get sacid from the other on-station sources). It's not about preventing random players from using it so much as demanding they put in at least a little work for it.

**Comment by Eneocho** [comment]
> 
> 
> > I think for consistency the sacid tank should come in a large box just like the welder fuel tank instead of a renamed shardcrate. You can "acquire" sacid from engi and chem, so it ain't like the shardcrate will deter sacid memery.
> 
> Locked cargo crates are never a deterrent to a determined player, but they at least require you to put in a token effort to open them (just like it takes at least a token effort to get sacid from the other on-station sources). It's not about preventing random players from using it so much as demanding they put in at least a little work for it.

My point was that the (comparatively) more dangerous crate with a fuel tank can be opened with a crowbar, so why limit the sacid tank behind access? Consistency and all, man. All tanks come in wooden boxes, this one should, too.

**Comment by Apogee-dev** [comment]
> My point was that the (comparatively) more dangerous crate with a fuel tank can be opened with a crowbar, so why limit the sacid tank behind access? Consistency and all, man. All tanks come in wooden boxes, this one should, too.

I unironically think sacid has more griff potential than a fuel tank does, in part because setting off a welderbomb as a nonantag can and will get someone banned but melting masks and equipment is more of a grey area. That being said, yeah, I was waffling a bit on it when I went ahead and put it in the locked box. If more players think it should go in a crate I'll change it back, but I want to err on the side of caution for now.

**Comment by Eneocho** [comment]
You can already acquire sacid from tons of places that don't require you to ask cargo or chem for some acid pretty please, you can: 
-Electrolyze cream for milk and sacid.
-Grind some ZAM food (from the cargo merch store) for nutriment and sacid.
-Grind a flare then mix with water and electrolized oxygen for pure sacid.
-Gamble with Groans Soda for the banned: sour surprise for pacid, which is worse. 
-Break into engineering and steal the sacid bottles from the Engi-Vend.
-Break into RnD and make a new chem dispenser. Or stealing the one already there.
-Plant some nettle or deathnettle seeds for sacid and pacid respectively.
-Break into chem and steal a chem dispenser or fill a few beakers with some sacid.
Point is that hiding the sacid tank inside a locked box won't slow down the guy that wants it, given that there are tons of other ways to acquire some. And even with all of these already available, sacid memery/griff ain't really a thing.

**Comment by Apogee-dev** [comment]
> 
> 
> You can already acquire sacid from tons of places that don't require you to ask cargo or chem for some acid pretty please, you can:
> -Electrolyze cream for milk and sacid.
> -Grind some ZAM food (from the cargo merch store) for nutriment and sacid.
> -Grind a flare then mix with water and electrolized oxygen for pure sacid.
> -Gamble with Groans Soda for the banned: sour surprise for pacid, which is worse.
> -Break into engineering and steal the sacid bottles from the Engi-Vend.
> -Break into RnD and make a new chem dispenser. Or stealing the one already there.
> -Plant some nettle or deathnettle seeds for sacid and pacid respectively.
> -Break into chem and steal the chem dispenser.
> Point is that hiding the sacid tank inside a locked box won't slow down the guy that wants it, given that there are tons of other ways to acquire some. And even with all of these already available, sacid memery ain't really a thing.

Sure, and per my earlier point those all require at least a bit of effort or game knowledge, and they're all surreptitious in some way. They all also provide acid in much smaller quantities, though it's highly unlikely you'll ever use up a full tank of the stuff. It's not about preventing people from getting it or even about making it _hard_ to get, but it still ought to give all the usual signals that John McGreytide isn't supposed to be messing around with it in the open.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -309,6 +309,15 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "silicate tank crate"
 	group = "Supplies"
 
+/datum/supply_packs/sacidtank
+	name = "Sulphuric acid tank"
+	contains = list(/obj/structure/reagent_dispensers/sacid)
+	cost = 8
```
Comment (Kurfursten) [inline code/datums/supplypacks.dm:315]: That's a lot for 8 credits. I think the others were balanced around their commonness in price.

That said it's not a dealbreaker.

Comment (Apogee-dev) [reply code/datums/supplypacks.dm:315]: Yeah, I wasn't 100% on the price. I'll raise it later if we start seeing too much of these things.

**Pull request merged**